src/app/scheduler/scheduled-process/scheduled-process.component.ts

```diff
@@ -156,4 +156,66 @@ export class ScheduledProcessComponent implements OnInit {
   getISODay(index: number) {
     return weekDays[index];
   }
+
+  /**
+   * Disable schedule
+   * @param schedule_id id of the schedule to disable
+   */
+  public disableSchedule(schedule_id) {
+    console.log('Schedule ID:', schedule_id);
```

Comment:
remove this log

Reply:
no, but say console.log('Disabling Schedule :', schedule_id);

do the same for enable!